### PR TITLE
Task overview editable select style

### PR DESCRIPTION
**Pull request by AnttiVaarala**
task overview editable select on same line + textarea style fixes

### CODE REVIEW

File: frontend/src/components/EditableSelect.module.scss [hunk 1]
```diff
@@ -9,6 +9,15 @@
   margin-top: 0;
 }
 
+.editButtonDiv {
+  margin-top: 0;
```
Comment (IidaKainu) [inline frontend/src/components/EditableSelect.module.scss:13]: I did not see this affecting the styles (?). If that's correct, the line can be removed

**Review by IidaKainu — APPROVED** [review]
Looks good to go, but consider the comment below before merging 👍

**Comment by juhoeerola** [comment]
What is the status of this one?

The same changes seem to also be in #444

**Comment by AnttiVaarala** [comment]
It's the same, this was originally just for the editableSelect styling but there were many other changes for task overview so I added all the task overview changes to same branch, I'll close this

**Pull request closed without merging**